### ISSUE TITLE
Formatting toolbar gets moved to the top after edit is clicked to setup the Event detail

### ISSUE THREAD

**Issue by white3tj** [SparkDevNetwork/Rock#4707]
![formattingRibbonBug](https://user-images.githubusercontent.com/85329759/120746781-e38de880-c4cd-11eb-9f34-969a9e1e08a5.jpg)

### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Did you perform a search at https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ARock to see if your bug or enhancement is already reported?
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?

### Description
Formatting toolbar gets moved to the top after edit is clicked to setup the Event detail

_[Write a clear and concise description and include **full window** (non-partial) screenshots. FYI: You can just copy and paste an image directly into here and it will magically upload.]_ 

### Steps to Reproduce

1. Click this event page https://rock.vineyardcincinnati.com/page/396?EventCalendarId=1&EventItemId=5499
2. Scroll down and hit edit
3. You will see the Formatting toolbar appear at the top of the page, instead of above the description box

**Expected behavior:**

The formatting toolbar should sit above the description box to use

_[A clear and concise description of what you expected to happen.]_

**Actual behavior:**

Instead the Formatting toolbar moves to the top, underneath the search ribbon

_[A description of what actually happened.]_

### Versions

Rock Version: Rock McKinley 12.0 (1.12.0.10)

Client Culture Setting: en-US

* **Rock Version:** [Enter Rock version here]
* **Client Culture Setting:** [Enter Client Culture here]

**Comment by bjacky** [comment]
Running Rock 11.2, I am seeing the same issue occur on occasion with our instance of Rock but in our case the bar is disappearing, hidden behind the top navbar but it is not consistently a problem. When I inspect the code on the page I can see it is adding extra styling to the toolbar (element. style { position: fixed; top: 0px; width: 1736px; z-index:1000; } ) which puts it up at the top under the top navbar, out of view.

**Comment by bjacky** [comment]
I'm bypassing this issue at the moment by adding the following code to the Calendar Event Item Detail Pre-HTML to stop the repositioning issue that setting it to fixed is creating...
```
<style>
    .note-toolbar {
        position: relative !important; 
    }
</style>
```

**Comment by KelleyLangkamp** [comment]
Do either of you have an event attribute of the field type image? We've noticed that if we remove the image that is stored as an event attribute the toolbar comes back to it's right spot. Once the image is added to the event again, the toolbar moves. This is on Rock 12.3.

**Comment by crayzd92** [comment]
This is resolved with fixes introduced in #4456
